service/src/main/java/uk/gov/hmcts/reform/fpl/service/CommonCaseDataExtractionService.java

```diff
@@ -14,6 +14,7 @@
 
 import static org.apache.commons.lang3.StringUtils.defaultIfBlank;
 import static uk.gov.hmcts.reform.fpl.service.CaseDataExtractionService.EMPTY_PLACEHOLDER;
+import static uk.gov.hmcts.reform.fpl.service.CaseDataExtractionService.HEARING_EMPTY_PLACEHOLDER;
```

Comment:
I would suggest moving `HEARING_EMPTY_PLACEHOLDER` to either this class or `DocmosisTemplateDataGeneration`, think those are more logical places for it